### ISSUE TITLE
Sdr and Std are NaN when using guitar/others setup with moisesdb dataset

### ISSUE THREAD

**Issue by chenCFD** [ZFTurbo/Music-Source-Separation-Training#196]
Hi, thanks for your great repo!
I modified the config from configs/config_musdb18_mel_band_conformer.yaml.
The only change is in instruments, I changed it to:

instruments:
  - guitar
  - others

I used the moisesdb dataset, where I extracted guitar.wav and merged all other instruments into others.wav.

But when I start training, already in the first epoch I got NaN for SDR.
Here is the terminal log:

Found tracks in dataset: 320
Instruments: ['guitar', 'others']
Losses for training: ['masked_loss']
Metrics for training: ['sdr']. Metric for scheduler: sdr
Patience: 2 Reduce factor: 0.95
Batch size: 7 Grad accum steps: 1 Effective batch size: 7
Dataset type: 1
Optimizer: adam
Train for: 1000 epochs
Train epoch: 0 Learning rate: 5e-05
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 1000/1000 [09:59<00:00,  1.67it/s, loss=74.6, avg_loss=60.3]
Training loss: 0.6032469095438718
0it [00:00, ?it/s]
Num overlap: 4
Instr guitar sdr: nan (Std: nan)
Elapsed time: 0.03 sec
Train epoch: 1 Learning rate: 5e-05

Do you have any idea what could cause the NaN?
Thanks in advance! 🙏

**Comment by chenCFD** [comment]
Thanks for your remind! The problem has been resolved.
